components/Counting.vue

```diff
@@ -1,95 +1,192 @@
 <template>
     <section class="counting">
-      <h4>AI Insights delivered & counting </h4>
-      <h2 id="countUp">1234</h2>
+      <div id="countPhrase" >
+        <h4 id="countPhraseWrapper" >AI Insights delivered & counting</h4>
+      </div>
+      <div id="countUp">
+        <h2 id="countUpWrapper" > </h2>
+      </div>
     </section>
 </template>
 
 <script>
 export default {
     mounted() {
-    function animateValue(id, start, end, duration) {
-      var obj = document.getElementById(id);
-      var range = end - start;
-      var minTimer = 50;
-      var stepTime = Math.abs(Math.floor(duration / range));
-
-      stepTime = Math.max(stepTime, minTimer);
-
-      var startTime = new Date().getTime();
-      var endTime = startTime + duration;
-      var timer;
+      var canv = document.createElement('canvas');
+      var ctx = canv.getContext('2d');
+      var container = document.getElementById('countUpWrapper');
+      var previous_length = null;
+      var initial_font_size = 14.5; // vh
+      var size_tolerance_min = 0.2;
+      var size_tolerance_max = 0.12;
+      function fitTextToContainer(str) {
```

Comment:
we should be run on any `window.resize`, otherwise you'll see something like:

![image](https://user-images.githubusercontent.com/134478/57656827-2440ff00-759f-11e9-9357-71cc9fac7b65.png) _after resize_ 
![image](https://user-images.githubusercontent.com/134478/57656842-302cc100-759f-11e9-88ed-d3dfa2af95f7.png)

Reply:
Running on `window.resize` can be inconsistent across devices, but the thought was that only developers would be resizing. Orientation changes will occur though, and should trigger `resize`. I'll add a hook to resize and see how it behaves.

Reply:
yes. reorientation is what I'm more concerned about